bin/save_gradle_dependency_cache

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+
+# The key is shared with bin/restore_gradle_dependency_cache
```

Comment:
```suggestion
# The key is shared with `bin/restore_gradle_dependency_cache`
```

he backticks are for consistency with the comment in line 16 below, where script names are surrounded with backticks.